dev_guide/builds.adoc

```diff
@@ -515,8 +515,31 @@ endif::[]
 You can also manage environment variables defined in the `*BuildConfig*` with the
 xref:../dev_guide/environment_variables.adoc#overview[`oc set env`] command.
 
-[[custom-strategy-options]]
+[[adding-secrets-to-docker-strategy-build-configs]]
+=== Adding Secrets to Docker Strategy Build Configurations
+
+Add a secret to your build configuration so that it can access a private
+repository.
+
+. Create a new {product-title} project.
+
+. Create several secrets.
```

Comment:
Not sure if this wording is right. I would say something like: `Create secret that contains credentials for accessing private repository` .. Maybe even link secret creation process(if we got doc for it)